ui/src/app/analytics/dashboard/dashboard.component.html

```diff
@@ -1,152 +1,179 @@
-<button type="button" (click)="resetDashboard()"
-        class="btn btn-default">Reset <span class="glyphicon glyphicon-trash"></span></button>
```

Comment:
Gone after refactoring. Is it on purpose? Feels like it might be, but thought I'd ask to be sure...